docs/revsets.md

```diff
@@ -288,6 +288,8 @@ revsets (expressions) as arguments.
 * `author_date(pattern)`: Commits with author dates matching the specified [date
   pattern](#date-patterns).
 
+* `signed()`: Commits that are cryptographically signed.
```

Comment:
nit: can you move this after `committer_date()` or before `author()`?
`author*()` and `committer*()` functions are quite similar.